yarn.lock

```diff
@@ -3240,7 +3240,7 @@ ethereum-cryptography@^0.1.3:
 
 "ethereumjs-abi@git+https://github.com/ethereumjs/ethereumjs-abi.git":
   version "0.6.8"
-  resolved "git+ssh://git@github.com/ethereumjs/ethereumjs-abi.git#1a27c59c15ab1e95ee8e5c4ed6ad814c49cc439e"
+  resolved "git://git@github.com/ethereumjs/ethereumjs-abi.git#1a27c59c15ab1e95ee8e5c4ed6ad814c49cc439e"
```

Comment:
what a move! might need to be `https://github.com/ethereumjs/ethereumjs-abi#1a27c59c15ab1e95ee8e5c4ed6ad814c49cc439e`

Reply:
or otherwise mimic https call https://github.com/BanklessHQ/bankless-dao-website/pull/13/commits/f769b1b2853f085ddc6a419479c3eaf7620acc0b#diff-51e4f558fae534656963876761c95b83b6ef5da5103c4adef6768219ed76c2deR3222